code/modules/surgery/hepatectomy.dm

```diff
@@ -0,0 +1,50 @@
+/datum/surgery/hepatectomy
+	name = "Hepatectomy"
+	target_mobtypes = list(/mob/living/carbon/human, /mob/living/carbon/monkey)
+	possible_locs = list(BODY_ZONE_CHEST)
+	requires_real_bodypart = 1
+	steps = list(/datum/surgery_step/incise,
+		/datum/surgery_step/retract_skin,
+		/datum/surgery_step/saw,
+		/datum/surgery_step/clamp_bleeders,
+		/datum/surgery_step/incise,
+		/datum/surgery_step/hepatectomy,
+		/datum/surgery_step/close
+		)
+
+/datum/surgery/hepatectomy/can_start(mob/user, mob/living/carbon/target)
+	var/obj/item/organ/liver/L = target.getorganslot(ORGAN_SLOT_LIVER)
+	if(L.damage > 50 && !(L.organ_flags & ORGAN_FAILING))
+		return TRUE
+	else
+		return FALSE
+
+////hepatectomy, removes damaged parts of the liver so that the liver may regenerate properly
+//95% chance of success, not 100 because organs are delicate
+/datum/surgery_step/hepatectomy
+	name = "remove damaged liver section"
+	implements = list(/obj/item/scalpel = 95, /obj/item/melee/transforming/energy/sword = 65, /obj/item/kitchen/knife = 45,
+		/obj/item/shard = 35)
+	time = 52
+
+/datum/surgery_step/hepatectomy/preop(mob/user, mob/living/carbon/target, target_zone, obj/item/tool, datum/surgery/surgery)
+	display_results(user, target, "<span class='notice'>You begin to cut out a damaged peice of [target]'s liver...</span>",
+		"<span class='notice'>[user] begins to make an incision in [target].</span>",
+		"<span class='notice'>[user] begins to make an incision in [target].</span>")
+
+/datum/surgery_step/hepatectomy/success(mob/user, mob/living/carbon/target, target_zone, obj/item/tool, datum/surgery/surgery)
+	if(ishuman(target))
```

Comment:
Why the ishuman check?

Reply:
i took reused that from coronary bypass and lobectomy

Reply:
Well, you literally added this ``	target_mobtypes = list(/mob/living/carbon/human, /mob/living/carbon/monkey)
`` at the top of your surgery.
So remove this check and also remove the ``/mob/living/carbon/monkey``
OR
Just remove the check.
Both cases are fine.